src/app.ts

```diff
@@ -434,11 +432,13 @@ export function newApp({
             serviceProviderConfig
           }).run(),
         err => new Error(`Unexpected error initizing Spid Login: [${err}]`)
-      );
+      ).map(withSpidApp => ({ ...withSpidApp, timer }));
```

Comment:
If you pass the item through the pipe, you don't even need a `TimeTracer` object, you can just pass the time value.
The whole point of `TimeTracer` is to encapsulate its state so it can be used as a simple statement at any stage of the pipe, regardless of the steps between or even if they happen at a different nesting level.

So I suggest we either use plain bigint values or we declare `timer` in the upper scope.

Reply:
Fixed in 5ba62d5